config.yml

```diff
@@ -0,0 +1,9 @@
+default:
+  default_key: default_value
+
+python:
+  participants_csv_path: "/sbgenomics/project-files/i2b2_exported_participant_ids_2024-01-18_142546_9276.csv"
```

Comment:
Do we need to change/update this path for every new archive version?

Reply:
Ideally, the path of this manifest file would remain unchanged, only the contents within (`Participant Identifier`, `cohort`) should change.